README.md

```diff
@@ -311,7 +311,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [macvendors.co](https://macvendors.co/) - Get vendor name of your network device using its mac address
 
 ### Credentials
-
+- [WhiteIntel](https://whiteintel.io/) - Check if a company was a victim of stealer malwares.
```

Comment:
Usually I put new items at the end of the section list

Reply:
Alright sorry for hat